ros2_ws/src/rcdt_franka/test/conftest.py

```diff
@@ -0,0 +1,81 @@
+# SPDX-FileCopyrightText: Alliander N. V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Iterator
+
+import pytest
+import rclpy
+from launch.actions import IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch_ros import actions
+from rcdt_utilities.launch_utils import get_file_path
+from rclpy.executors import SingleThreadedExecutor
+from rclpy.node import Node
+
+
+@pytest.fixture(scope="module")
+def singleton_node() -> Iterator[Node]:
+    """Fixture to create a singleton node for testing."""
+    rclpy.init()
+    node = Node("test_node_singleton")
+    executor = SingleThreadedExecutor()
+    executor.add_node(node)
+    yield node
+    executor.remove_node(node)
+    node.destroy_node()
+    rclpy.shutdown()
+
+
+@pytest.fixture(scope="module")
+def controllers_launch() -> IncludeLaunchDescription:
+    """Fixture to create launch file for controllers."""
+    return IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            get_file_path("rcdt_franka", ["launch"], "controllers.launch.py")
+        )
+    )
```

Comment:
```suggestion
    return IncludeLaunchDescription(
        get_file_path("rcdt_franka", ["launch"], "controllers.launch.py")
    )
```

You don't need PythonLaunchDescriptionSource.